src/libmongoc/src/mongoc/mongoc-client-private.h

```diff
@@ -30,10 +30,16 @@
 #include "mongoc-read-prefs.h"
 #include "mongoc-rpc-private.h"
 #include "mongoc-opcode.h"
+
```

Comment:
```suggestion
```

Suggest removing unneeded whitespace changes to reduce diff. It may add some effort when investigating git history.